model/trace/gen-ai.yaml

```diff
@@ -0,0 +1,69 @@
+groups:
+  - id: gen_ai.request
```

Comment:
We don't need to update this now. But in the future, we'll want groups like this:

* `gen_ai.request` will be an "attribute group" with the common attributes relevant to all spans, including attributes like `gen_ai.system` and `gen_ai.request.model`.
* `gen_ai.request.llm` would be a group specifically for LLM spans.
* `gen_ai.request.embeddings` would be a group specifically for embeddings spans.

Reply:
Makes sense!